source/_posts/2015-09-03-hiring-apprentices.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Hiring Apprentices"
+date: 2015-09-09 14:44:39 -0600
+comments: true
+categories: hiring, apprenticeship, mentoring, ruby
+---
+
+The [MX](http://mx.com/) apprenticeship has been a lot of work and fun so far.
+This blog post is the beginning of an effort to document what I've learned along the way.
+
+# The Goal
+
+Initially I got interested in the idea of an apprenticeship while listening to the [ruby rogues episode with Joe Mastey and Jill Lynch](https://devchat.tv/ruby-rogues/190-rr-apprenticeship-with-joseph-mastey-and-jill-lynch-of-enova).
+I help to organize a [local ruby meetup group](http://utruby.org/#uv.rb) so I liked the idea of helping new engineers.
+I also love my job at MX so I liked the idea of helping to grow our team.
+
+When I discussed the idea of an apprenticeship with people at MX, the first thing I had to do was nail down what our objective was. "Helping new engineers" wasn't going to cut it.
+
+We decided that the primary objective was to __add amazing engineers to the team__.
+
+# Amazing Apprentices
+
+Based on my experience in the local Ruby community I believed that there were some really amazing programmers who were self-taught or graduating from bootcamps that were struggling to find first jobs.
+
+The shortage of senior engineers at pretty much every company has made it really difficult to get a job as a junior developer. Lots of companies think that they can't afford to hire junior engineers because they don't have enough senior people to train them.
+There are also serious bias issues throughout the software industry and especially in the hiring processes.
```

Comment:
I've been thinking about this sentence. And I'm not sure what I want to do about it. If I name one or a few specific examples of bias, I think I need to then cite some source(s) for that information. It would take a good amount of work to find a source worth citing on this so that I feel comfortable defending the position. I'm also a little worried that it takes away from the focus of my post. I don't want readers to get lost thinking about the controversial debates around privilege and bias in software.

On the other hand if I leave it ambiguous, it makes it more comfortable for the reader to skim over it, but definitely makes it less direct and meaningful.